pom.xml

```diff
@@ -399,32 +397,134 @@
             <version>2.4.12</version>
             <scope>provided</scope>
         </dependency>
-        <!--Tests-->
-        <dependency>
-            <groupId>org.jenkins-ci.main</groupId>
-            <artifactId>jenkins-test-harness</artifactId>
-            <version>2.44</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
             <version>5.3.2</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins.workflow</groupId>
-            <artifactId>workflow-aggregator</artifactId>
-            <version>2.0</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.jfrog.artifactory.client</groupId>
             <artifactId>artifactory-java-client-services</artifactId>
             <version>2.6.2</version>
             <scope>test</scope>
         </dependency>
         <!--/Tests-->
+        <!--- Upper bound dependencies -->
```

Comment:
Shouldn't the upper bound restrictions not just be placed inside `<dependencyManagement>`?